scripts/ci/create-docker-image.sh

```diff
@@ -29,9 +30,28 @@ GOBUILD_DIR=/root/.cache/go-build
 TOPLEVEL_VOL=$PWD
 TOPLEVEL_DIR=/root/go/src/github.com/skydive-project/skydive
 
+LAST_VERSION=$( git describe --abbrev=0 --tags | sed -e s/v// )
+LAST_COMMIT=$( git rev-parse --short=12 HEAD )
+DOCKER_TAG_SNAPSHOT=${LAST_VERSION}-${LAST_COMMIT}
```

Comment:
If you have only the SHA1, we need to find the version that corresponds to the SHA1. We can only use the commit

Reply:
Would the HEAD in that case not be set to the SHA1 value we get? If so then I think it should still work:

```
[vagrant@dev ci]$ git checkout v0.20.0
Note: checking out 'v0.20.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 92e1c38a Merge pull request #1364 from lebauce/v0.20.0
[vagrant@dev ci]$ git describe --abbrev=0 --tags
v0.20.0
```